packages/wallet/api/src/lib-wallet.js

```diff
@@ -523,7 +524,7 @@ export function makeWalletRoot({
     inboxState.set(id, offerForDisplay);
     if (doPush) {
       // Only trigger a state change if this was a single update.
-      offersUpdater.updateState([...inboxState.values()]);
+      offersUpdater.updateState([offerForDisplay]);
```

Comment:
this doesn't help with `updateAllInboxState`. Missed or deferred?

Reply:
um... deferred... as in... I'm not sure what that thing does... let's see if it's a problem in practice... moderate integration testing didn't turn up anything bad.